plugins/examples/painless-whitelist/build.gradle

```diff
@@ -30,8 +30,12 @@ dependencies {
   compileOnly project(':modules:lang-painless')
 }
 
-integTestCluster {
-  distribution = 'zip'
+if (System.getProperty('tests.distribution') == null) {
+  integTestCluster.distribution = 'oss-zip'
+}
+
+if (integTestCluster.distribution == 'integ-test-zip') {
```

Comment:
I think we can remove this line because we don't support setting the setting to this.

Reply:
We can add something to cluster formation tasks to make sure we don't set the system property to `integ-test-zip`.